Judge/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/event/PlayerShootResultEventListener.java

```diff
@@ -0,0 +1,43 @@
+package pl.jakubowskiprzemyslaw.tajgertim.event;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationListener;
+import org.springframework.stereotype.Component;
+import pl.jakubowskiprzemyslaw.tajgertim.models.round.NextRoundStatus;
+import pl.jakubowskiprzemyslaw.tajgertim.models.round.RoundStatus;
+import pl.jakubowskiprzemyslaw.tajgertim.models.shoot.PlayerShootResult;
+import pl.jakubowskiprzemyslaw.tajgertim.models.shoot.ShootResult;
+import pl.jakubowskiprzemyslaw.tajgertim.queues.Queues;
+import pl.jakubowskiprzemyslaw.tajgertim.services.QueueService;
+
+@Component
+public class PlayerShootResultEventListener implements ApplicationListener<PlayerShootResultEvent> {
+
+    private final QueueService queueService;
+
+    @Autowired
+    public PlayerShootResultEventListener(QueueService queueService) {
+        this.queueService = queueService;
+    }
+
+    @Override
+    public void onApplicationEvent(PlayerShootResultEvent event) {
+        PlayerShootResult playerShootResult = event.getPlayerShootResult();
+        ShootResult shootResult = playerShootResult.getShootResult();
+
+        RoundStatus roundStatus;
+        switch (shootResult) {
```

Comment:
I suggest that this switch would be better in some private method below.

Reply:
done